walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/Event.java

```diff
@@ -93,4 +94,14 @@ private static boolean equals(Event a, Event b) {
     // interface documentation, equals will check for set-equality across all set implementations.
     return a.title.equals(b.title) && a.when.equals(b.when) && a.attendees.equals(b.attendees);
   }
+
+  /**
+   * Enable sort by the start time.
+   */
+    public static final Comparator<Event> SORT_BY_START_TIME = new Comparator<Event> () {
```

Comment:
In ascending order of descending order? It'd be clear to mention it in the name and comment.

Reply:
The variable name is updated!